xml/admin_monitoring_alerting.xml

```diff
@@ -195,7 +164,61 @@
    </para>
 <screen>&prompt.cephuser;ceph config rm mgr mgr/cephadm/container_image_prometheus</screen>
   </note>
+
+ </sect1>
+
+ <sect1 xml:id="monitoring-applying-updates">
+  <title>Updating monitoring services</title>
+  <para>
+    As mentioned in <xref linkend="monitoring-custom-images" />, &cephadm; is
+    shipped with the locations to the recommended and tested container images. These are
+    the ones that are used by default, too.
+  </para>
+  <para>
+    By updating the &ceph; packages, new versions of these locations may be
+    shipped. This just updates where the container images are pulled from, it
+    does not update any services.
+  </para>
+
+  <para>
+    After the locations to the new container images have been updated, either
+    manually, as described in <xref linkend="monitoring-custom-images" />, or
+    automatically through an update of the &ceph; package, the monitoring
+    services can be updated.
+  </para>
+
+  <para>
+    To do so, use <literal>ceph orch redeploy</literal> like so:
+  </para>
+
+  <screen>
+&prompt.cephuser;ceph orch redeploy node-exporter
+&prompt.cephuser;ceph orch redeploy prometheus
+&prompt.cephuser;ceph orch redeploy alertmanager
+&prompt.cephuser;ceph orch redeploy grafana
+  </screen>
+
+  <para>
+    Currently no single command to update all monitoring services exist. The
+    order in which these services are updated is generally not important.
+  </para>
+
+  <note>
+    <para>
+      If you use custom container images, the locations specified for the
+      monitoring services won't change automatically if the &ceph; packages are
+      updated. If you have specified custom container images, you will need to
+      specify the locations to the new container images manually. This may be
+      the case if you use a local container registry.
```

Comment:
```suggestion
      If you use custom container images, the URLs specified for the
      monitoring services will not change automatically if the &ceph; packages are
      updated. If you have specified custom container images, you will need to
      specify the URLs of the new container images manually. This may be
      the case if you use a local container registry.
```